backend/src/api/transaction-utils.ts

```diff
@@ -74,8 +74,18 @@ class TransactionUtils {
 
   public async $getMempoolTransactionsExtended(txids: string[], addPrevouts = false, lazyPrevouts = false, forceCore = false): Promise<MempoolTransactionExtended[]> {
     if (forceCore || config.MEMPOOL.BACKEND !== 'esplora') {
-      const results = await Promise.allSettled(txids.map(txid => this.$getTransactionExtended(txid, addPrevouts, lazyPrevouts, forceCore, true)));
-      return (results.filter(r => r.status === 'fulfilled') as PromiseFulfilledResult<MempoolTransactionExtended>[]).map(r => r.value);
+      const results: MempoolTransactionExtended[] = [];
+      for (const txid of txids) {
+        try {
+          const result = await this.$getMempoolTransactionExtended(txid, addPrevouts, lazyPrevouts, forceCore);
+          if (result) {
+            results.push(result);
+          }
+        } catch {
+          // skip failures
```

Comment:
Would be good to log something in DEBUG, or do we already log in another function?

Reply:
Also for the dev who will land on this comment in a few months, could be good to elaborate a bit why we're fine skipping failures

Reply:
we already log progress in the outer function (although those logs might be a bit sparse with a slow core backend)